pkg/config/config.go

```diff
@@ -287,9 +288,21 @@ func DataDir() string {
 	return path
 }
 
-// CacheDir returns the default path for gh cli cache.
+// Cache path precedence: XDG_CACHE_HOME, LocalAppData (windows only), HOME.
```

Comment:
Is it worth adding the lowest precedent?

```suggestion
// Cache path precedence: XDG_CACHE_HOME, LocalAppData (windows only), HOME, legacy gh-cli-cache.
```